docs/kickstart.rst

```diff
@@ -6,8 +6,8 @@ Anaconda Kickstart Documentation
 
 Anaconda uses `kickstart <https://github.com/rhinstaller/pykickstart>`_ to automate
 installation and as a data store for the user interface. It also extends the kickstart
-commands `documented here <https://pykickstart.readthedocs.io/>`_
-by adding a new kickstart section named ``%anaconda`` where commands to control the behavior
+commands `documented here <https://github.com/rhinstaller/pykickstart/blob/master/docs/kickstart-docs.rst>`_
+by adding a new kickstart section named %anaconda where commands to control the behavior
```

Comment:
Why no to use backtricks formating for special/reserved words ? Looks like you did not run `git pull` here :)